### PR TITLE
ci: enable ipv6 test

### PR DESCRIPTION
**Pull request by zirain**

### CODE REVIEW

File: test/config/gatewayclass.yaml [hunk 1]
```diff
@@ -16,6 +16,7 @@ metadata:
   name: proxy-config
   namespace: envoy-gateway-system
 spec:
+  ipFamily: IPv4
```
Comment (zirain) [inline test/config/gatewayclass.yaml:19]: IpFamily needs to be set explicitly, `sed` will change it to right value.

Comment (zhaohuabing) [reply test/config/gatewayclass.yaml:19]: Nit: use `{IP_FAMILY}` instead of `IPv4` to make it clearer?

Comment (zirain) [reply test/config/gatewayclass.yaml:19]: use Ipv4 directly allow you apply this file without run `make prepare-ip-family` for debugging

**Review by arkodg — APPROVED** [review]
LGTM thanks for driving this !

**Review by zhaohuabing — APPROVED** [review]

**Pull request merged**